src/pages/PointOfContact/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/require-default-props */
```

Comment:
Want to just disable this rule for now?

See: https://github.com/cfpb/sbl-frontend/issues/355

Reply:
@billhimmelsbach Pulled that ticket into the Sprint, but will probably make the change next sprint.  And will remove all of these `disable`s as part of that work.